great_expectations/profile/user_configurable_profiler.py

```diff
@@ -770,14 +792,31 @@ def _build_expectations_numeric(self, dataset, column, **kwargs):
                 )
 
         # quantile values
+        # if (
+        #     "expect_column_quantile_values_to_be_between"
+        #     not in self.excluded_expectations
+        # ) and isinstance(dataset, Dataset):
         if (
             "expect_column_quantile_values_to_be_between"
             not in self.excluded_expectations
         ):
             if isinstance(dataset, PandasDataset):
-                allow_relative_error = "lower"
-            else:
-                allow_relative_error = dataset.attempt_allowing_relative_error()
+                if isinstance(dataset, PandasDataset):
```

Comment:
Shouldn't the outer `if` statement be `isinstance(dataset, PandasDataset)`? Otherwise, a bit confused by repeat.

Reply:
Great catch - this was a typo - outer `if` should be `isinstance(dataset, Dataset)`

Reply:
Haha. It looks like I made the same typo. :) I meant to say `isinstance(dataset, Dataset)`.